chall12/vde-dios.c

```diff
@@ -0,0 +1,55 @@
+#include <string.h>
+
+void    swap_str(char *str, int start, int finish)
+{
+    char        aux;
+    int         i;
+
+    i = start + 1;
+    while (i < (finish - start + 1) / 2 + start)
+    {
+        aux = str[i];
+        str[i] = str[finish + start - i];
+        str[i] == ')' ? str[i] = '(' : (str[i] == '(' ? str[i] = ')' : str[i]);
+        aux == ')' ? aux = '(' : (aux == '(' ? aux = ')' : aux);
+        str[finish + start - i] = aux;
+        i++;
+    }
+    if ((finish - start + 1) % 2)
+        str[i] == ')' ? str[i] = '(' : (str[i] == '(' ? str[i] = ')' : str[i]);
```

Comment:
You could have swapped while copying to avoid writing the same string over over and over ;)

Reply:
I have been thinking about how to swap the strings without using an aux var and haven't got to any solution... What do you mean with this?